tests/integration/tier0/non-destructive/enabled-repositories-after-analysis/main.fmf

```diff
@@ -0,0 +1,20 @@
+summary: |
+    Verify repositories stay enabled after analysis
+
+tag+:
+    - non-destructive
+    - satellite
+
```

Comment:
```suggestion
enabled: false
adjust+:
    enabled: true
    when: distro == centos-7
    because: currently we only have satellite key for CentOS7
```